client/ayon_core/hosts/max/plugins/load/load_max_scene.py

```diff
@@ -14,6 +16,56 @@
 from ayon_core.pipeline import get_representation_path, load
 
 
+class MaterialDupOptionsWindow(QtWidgets.QDialog):
+
```

Comment:
Can we add a simple rudimentary docstring for this?

Future readers of the code will love you.